### DISCUSSION TITLE
How to plan by units and capacity

### DISCUSSION THREAD

**Discussion by kobsam** [frePPLe/frepple#547] [Q&A]
Hi, I need to plan the date of my manufactoring-orders not by the duration of the operation, but by units. Its just a rough planning to get the day and the resource of the manufactoring-order. The duration of the operation is 1 day and the calculated start-time is just the day where it gets produced. So I have eg a workcenter (my resource) which can produce 1000 units a day. Then I have my demands with the deserved quantity. I saw the resource_buckets for that problem in the documentation, but I did not find any example that satisfy my problem. How can I specify that the algorithm is planning by units, specify the max-units for a day and define that it should plan the operation earlier if no more capacity is available?

Thank you very much!

**Comment by jdetaeye** [comment]
Resource of type 'buckets' are indeed what you need.

Check out this example https://frepple.org/docs/current/user-guide/examples/resource/resource-type.php and the online data https://demo.frepple.com/resource-type/resource/
